src/reynir_correct/spelling.py

```diff
@@ -690,25 +717,39 @@ def correct(
         context: Tuple[str, ...] = (),
         at_sentence_start: bool = False
     ) -> str:
-        """ Correct a single word, keeping its case (lower/upper/title) intact.
-            The optional context parameter contains a tuple of preceding
-            words, used to enable a more accurate probability prediction. """
+        """Correct a single word, keeping its case (lower/upper/title) intact.
+        The optional context parameter contains a tuple of preceding
+        words, used to enable a more accurate probability prediction."""
         return self._case_of(word)(
             self._correct(word, self._cast(word), context, at_sentence_start)
         )
 
+    def suggest_list(
+        self,
+        word: str,
+        *,
+        context: Tuple[str, ...] = (),
+        at_sentence_start: bool = False
+    ) -> List[Tuple[str, float]]:
+        """Return a list of suggestions for a single word, keeping its case
+        (lower/upper/title) intact. The optional context parameter contains
+        a tuple of preceding words, used to enable a more accurate probability
+        prediction."""
+        x = self._best_candidates(word, self._cast(word), context, at_sentence_start)
```

Comment:
The variables x and y could be given more meaningful names

Reply:
Done